app/containers/Dashboard/index.js

```diff
@@ -119,9 +120,13 @@ DashboardRoot.propTypes = {
   web3Redux: PropTypes.any,
 };
 
-const mapDispatchToProps = () => ({
-  setActiveTab,
-});
+function mapDispatchToProps() {
+  return {
+    setActiveTab,
+    modalDismiss,
+    modalAdd,
```

Comment:
Root dashboard container is the bad place for passing modal actions. It does not require it, so will be better to bind this actions in subcontainers (in Wallet for example).

Reply:
OK, will change, too